src/netlog/netlog-network.c

```diff
@@ -149,7 +149,11 @@ int manager_push_to_network(Manager *m,
 
         IOVEC_SET_STRING(iov[n++], " ");
 
-        /* Ninth: message */
+        /* Ninth: message (truncated to 1000 bytes to fit within MTU) */
+        if (strlen(message) >= 1000) {
+            ((char*)message)[1000] = 0;
+        }
```

Comment:
Well this is precisely fit to your use case not a generic .  Rather If you make it configurable that would be nice  Or something like `MTU=`

Reply:
Or may be the interface name and let netlod figure out the maximum log length.

Reply:
Also, can we please have a warning message when this truncation happens? Now that we have TCP support, there exists something the user can reasonably do about this problem (switch from UDP to TCP). So it makes sense to warn them. But maybe only once per run - we don't want to output a warning for every single packet.